drivers/bluetooth/hci/hci_nxp.c

```diff
@@ -137,6 +166,54 @@ static int bt_nxp_set_calibration_data_annex100(void)
 
 #endif /* CONFIG_HCI_NXP_SET_CAL_DATA */
 
+#if CONFIG_BT_HCI_SET_PUBLIC_ADDR
```

Comment:
This should be `#ifdef` (or `#if defined(...)`) and not `#if`. However, if you look at my other comment maybe you can just remove it completely.

Reply:
Changed `#if` to `#if define()`. Thanks